base/build.gradle

```diff
@@ -85,6 +85,13 @@ dependencies {
     //api 'io.github.adempiere:tools:'
 }
 
+configurations.all {
```

Comment:
Please remove this change and create a new pull request only for this. Is more clean

Reply:
These changes are necessary for the file handler to work.

Reply:
Make other pull request with these changes

Reply:
The requested changes were implemented

Reply:
The requested changes were implemented